uwblueprint/community-fridge-kw
# fix: update calendar and fix reccurring end date input

## Brief description. What is this change? 

### [Wrong end date in form after editing](https://www.notion.so/uwblueprintexecs/Wrong-end-date-in-form-after-editing-81832748814d4b0f8a6784f4fdfa3fbf)

### [Disable ability to choose past dates](https://www.notion.so/uwblueprintexecs/Upcoming-Donations-Calendar-Disability-ability-to-choose-past-dates-642aca774d4648fb8e715cae3c6a06cc)



## Implementation description. How did you make this change?
* Fix reccurring end date input to display correct date
* Add `minDate` to calendar to prevent user from clicking past dates



## Steps to test
1. Schedule a donation - after clicking next, go back to first page to make sure reccurring end date field is correct
2. Check same field when editing the donation

1. Check calendar on homepage




## Checklist
